### PR TITLE
[JIT] Enable NVFuser tests in CI

### LINKED ISSUE #72466: Add nvfuser tests to CI
### 🚀 The feature, motivation and pitch

`test/test_jit_cuda_fuser.py` doesn't seem like it's in any github or internal CI runs. Check if this is true - if so, we should try to add it.

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by davidberard98**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #75116
* #71299
* __->__ #73322
* #74361

These tests have been disabled in OSS CI since #34785.

This disables the windows tests, which currently aren't passing.

Differential Revision: [D34436844](https://our.internmc.facebook.com/intern/diff/D34436844)

### CODE REVIEW

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Review by jjsjann123 — APPROVED** [review]
LGTM

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34436844).

**Review by eellison — APPROVED** [review]
Sweet! LGTM - @jjsjann123 may want to look over as well

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -588,7 +595,8 @@ def test_unary_ops(self):
             self._unary_test_helper(op, dtype, False)  # test special numbers
             self._unary_test_helper(op, dtype, True)  # test random data
 
-    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skipIf(True, "See issues 73395 and 75029")
```
Comment (jjsjann123) [inline test/test_jit_cuda_fuser.py:598]: hmmmm.. I don't remember seeing test failures in my last upstream push. let me take a look here.

**Review by jjsjann123 — APPROVED** [review]
code change looks fine. Let's merge this one.
I can double check the failing tests afterwards.

**Pull request closed without merging**